lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -367,12 +366,35 @@ export class UnifiedCacheManager implements ICacheManager {
      * returns a boolean if the given credential is removed
      * @param credential
      */
-    removeCredential(credential: Credential): boolean {
+    removeCredential(credential: CredentialEntity): boolean {
         const key = credential.generateCredentialKey();
-        return this.cacheStorage.removeItem(
-            key,
-            CacheSchemaType.CREDENTIAL,
-            this.inMemory
-        );
+        return this.removeItem(key, CacheSchemaType.CREDENTIAL);
+    }
+}
+
+export class DefaultStorageClass extends CacheManager {
```

Comment:
All default implementation are in ClientConfiguration. Seems like this should also go int there?

Reply:
I put this here because if the interface ever changes, you can make all the changes in a single file instead of having to jump between here and ClientConfiguration.ts